tests/amx/harness_amx.py

```diff
@@ -13,8 +13,10 @@
     pytest.skip("skipping AMX tests; could not find sde",
                 allow_module_level=True)
 
-CC_BAREMETAL        = os.getenv('CLANG') or os.getenv('CC', 'clang-13')
+# CC_BAREMETAL        = os.getenv('CLANG') or os.getenv('CC', 'clang-13')
+CC_BAREMETAL = "clang++"
 CFLAGS_BAREMETAL    = ' '.join([
+                        f'-std=c++11',
```

Comment:
No.

Reply:
Constexpr is supported from C++11

Reply:
This is okay for the _test code_ only, _not_ for code that is generated by SYSTL. This will need to be re-written to use two different compilers.